_layouts/post.html

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+---
+
+{% if page.image %}
+  {% unless page.image_hero == false %}
+    <div class="post-feature_image" style="background-image: url('{{ site.baseurl }}{{ page.image }}')"></div>
+  {% endunless  %}
+{% endif %}
+
+<div class="usa-grid">
+
+  <a class="back-blog link-arrow-left" href="{{ site.baseurl }}/blog/">Blog</a>
+  <article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+    <header class="post-header">
+      <h1 class="post-title" itemprop="name headline">{{ page.title }}</h1>
+      <p class="post-byline">
+      {% if page.authors %}
```

Comment:
Made it into an include in 1e47713. However, I don't know how to make it so it shows `{% if page.authors %}` and `{% if post.authors %}` for the different pages. So post.html uses `{% if page.authors %}`, while the tag page and blog index page uses `{% if post.authors %}`. Any idea how to do this?